test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -89,3 +90,13 @@ def test_pytorch_mpi(pytorch_training, ec2_connection, gpu_only, py3_only):
 @pytest.mark.parametrize("ec2_instance_type", PT_EC2_GPU_INSTANCE_TYPE, indirect=True)
 def test_nvapex(pytorch_training, ec2_connection, gpu_only):
     execute_ec2_training_test(ec2_connection, pytorch_training, PT_APEX_CMD)
+
+
+@pytest.mark.parametrize("ec2_instance_type", PT_EC2_GPU_INSTANCE_TYPE, indirect=True)
+def test_pytorch_telemetry_gpu(pytorch_training, ec2_connection, gpu_only):
+    execute_ec2_training_test(ec2_connection, pytorch_training, PT_TELEMETRY_CMD)
+
+
+@pytest.mark.parametrize("ec2_instance_type", PT_EC2_CPU_INSTANCE_TYPE, indirect=True)
```

Comment:
Can you run it in one instance type (https://github.com/aws/deep-learning-containers/pull/180/files#diff-8eba65ab07b750d3dcef3c76cbc30e82R89)